compiler/one-cmds/one-import-onnx

```diff
@@ -248,7 +248,7 @@ def _check_ext(args):
     if oneutils.is_valid_attr(args, 'disable_ext'):
         return None
     env_disable_ext = os.getenv('ONE_IMPORT_ONNX_EXT_DISABLE')
-    if env_disable_ext == '1' or env_disable_ext.upper() == 'Y':
+    if env_disable_ext == '1' or env_disable_ext == 'Y':
```

Comment:
we can check `env_disable_ext` is `None` or not but I want to go simple check with `1` or `Y`